### DISCUSSION TITLE
Malware and crypto notification

### DISCUSSION THREAD

**Discussion by Winxcornerquestion** [vhanla/winxcorners#47] [General]
When I ran this program through hybrid analysis it shows malware and crypto:
https://www.hybrid-analysis.com/sample/675ab42b6986cc831886d7c04cb07f2826183af1a6bd698d47f1308931acd4f2/64803c18d132022006030c68

1: [Found transport agent related strings used by exchange malware](https://www.hybrid-analysis.com/sample/675ab42b6986cc831886d7c04cb07f2826183af1a6bd698d47f1308931acd4f2/64803c18d132022006030c68#signature-43afea03de50484d16e37bc42570f557)

2: [Contains CRYPTO related strings](https://www.hybrid-analysis.com/sample/675ab42b6986cc831886d7c04cb07f2826183af1a6bd698d47f1308931acd4f2/64803c18d132022006030c68#signature-a95df205570b0783abe11578fa04ebfb)

3: [Sample was identified as malicious by at least one Antivirus engine](https://www.hybrid-analysis.com/sample/675ab42b6986cc831886d7c04cb07f2826183af1a6bd698d47f1308931acd4f2/64803c18d132022006030c68#signature-47e6a398cb360f5bbc8b712a661d4022)

**Comment by vhanla** [comment]
That's a false positive. Delphi pascal, assembler and other non popular compilers get this issue with those malware tools, unless the binary is signed, but that costs money. That's a common issue.
